### ISSUE TITLE
Long file names can't be selected via LCD

### ISSUE THREAD

**Issue by MBCook** [makerbot/G3Firmware#74]
Ran into an issue last night. I had a s3g file on my SD card with a name that was 26 characters long (not including the . or extension). When I browsed the SD card with the LCD I could see the file, but when you select it to build, nothing happens. It's like you never pressed the button. Other files worked fine.

I took the SD card back to my computer, renamed the file, and was able to select and start building from the LCD.

Original (bad) name: octopusv10block_fixed_sc055.s3g
Name that worked: octopus.s3g

I'm guessing the file name overran a buffer.

**Comment by eried** [comment]
Yeah, noticed that today too. I added page up & down support. I wish I can add folder support and some sorting options :D but I am going slowly.

Any idea about how to get your code merged? I don't understand how to send my pull request for page up & down support (never used github or any similar thing before)
